jekyll/_data/sidenavstyle.yml

```diff
@@ -0,0 +1,38 @@
+en:
+- name: Style
+  icon: icons/sidebar/getting_started.svg
+  children:
+    - name: Style and Voice
+      link: style/style-voice/
+    - name: Headings
+      link: style/style-headings/
+    - name: Links
+      link: style/style-links/
+    - name: Formatting
+      link: style/style-formatting/
+    - name: Using code samples
+      link: style/style-code/
+    - name: Using lists
+      link: style/style-lists/
+    - name: Using tables
+      link: style/style-tables/
+    - name: Using Screenshots
+      link: style/style-screenshots/
+    - name: Using Images
+      link: style/style-images/
+    
+
+ja:
+- name: Style
+  i18n_name: はじめよう
+  icon: icons/sidebar/getting_started.svg
+  children:
+    - name: Style and voice
+      i18n_name: 概要
+      link: ja/style/style-voice/
+    - name: CircleCI headings
+      i18n_name: 概要
+      link: ja/style/style-headings/
+    - name: CircleCI links
+      i18n_name: 概要
+      link: ja/style/style-links/
```

Comment:
This may not be necessary, but I translated it just in case. If you don't need it, ignore me🙈
- name: Style
  i18n_name: スタイル
  icon: icons/sidebar/getting_started.svg
  children:
    - name: Style and voice
      i18n_name: スタイルとボイス
      link: ja/style/style-voice/
    - name: Headings
      i18n_name: 見出し
      link: ja/style/style-headings/
    - name: Links
      i18n_name: リンク
      link: ja/style/style-links/
    - name: Formatting
      i18n_name: フォーマット
      link: style/style-formatting/
    - name: Using code samples
      i18n_name: コードサンプル
      link: style/style-code/
    - name: Using lists
      i18n_name: 箇条書き
      link: style/style-lists/
    - name: Using tables
      i18n_name: 表
      link: style/style-tables/
    - name: Using Screenshots
      i18n_name: スクリーンショット
      link: style/style-screenshots/
    - name: Using Images
      i18n_name: 画像
      link: style/style-images/

Reply:
Thanks for this, I will copy across!